src/validation/error.messages.ts

```diff
@@ -84,6 +87,7 @@ export enum ErrorMessages {
   SELECT_THE_TYPE_OF_MANAGING_OFFICER_YOU_WANT_TO_ADD = "Select the type of managing officer you want to add",
   SELECT_IF_SECURE_REGISTER_FILTER = "Select yes if any of the entity’s beneficial owners have ever applied to protect personal information at Companies House",
   SELECT_WHO_IS_MAKING_FILING = "Select who is completing this registration",
+  SELECT_WHO_IS_MAKING_FILING_UPDATE = "Select who is completing this update",
```

Comment:
SELECT_WHO_IS_MAKING_UPDATE_FILING to read better?

Reply:
Changed